### PR TITLE
feat: slider added

### PR DESCRIPTION
**Pull request by sumitch29**
### Slider added

- Simple slider
- styled slider
- slider with icon

Screenshot (408)
Screenshot (409)
Screenshot (410)

### CODE REVIEW

File: Components/slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+/* //////////styled slider///////// */
+
+.styled-slider {
+  appearance: none;
+  background: var(--light-gray);
+  cursor: pointer;
+  width: 15rem;
+  height: 5px;
+  border-radius: 5px;
```
Comment (vidhi-thakur) [inline Components/slider/slider.css:24]: Css variable could be used here

File: Components/slider/slider.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+            <h1>Styled Slider</h1>
+            <div class="box">
+                <div class="slider">
+                    <div class="slider-range">
+                        <p>10</p>
+                        <p>50</p>
+                        <p>100</p>
+                    </div>
+                    <input type="range" name="slider" class="styled-slider" id="" min="10" max="100" value="50">
```
Comment (vidhi-thakur) [inline Components/slider/slider.html]: `id` field is empty, if it is not required then you can remove it

**Review by ShoaibTamboli — COMMENTED** [review]
looks good to merge

File: Components/slider/slider.html (hunk 2 above)
Comment (vidhi-thakur) [reply Components/slider/slider.html]: I see same issue in other lines as well. Please check.

**Review by vidhi-thakur — APPROVED** [review]
Added a few suggestions, please check. Other than that, looks fine

File: Components/slider/slider.html (hunk 2 above)
Comment (sumitch29) [reply Components/slider/slider.html]: thanks for pointing our i will make changes

**Pull request merged**